xml/Makefile.am

```diff
@@ -21,7 +21,7 @@ dtddir			= $(CRM_DTD_DIRECTORY)
 dtd_DATA		= crm.dtd crm-transitional.dtd
 
 xsltdir			= $(dtddir)
-xslt_DATA		= upgrade06.xsl upgrade-*.xsl
+xslt_DATA		= upgrade06.xsl $(top_srcdir)/xml/upgrade-*.xsl
```

Comment:
I'm not sure "readlink" will work right in best-match.sh... you likely need realpath ...